test/tools/jsontests/BlockChainTests.cpp

```diff
@@ -650,7 +651,8 @@ void overwriteBlockHeaderForTest(mObject const& _blObj, TestBlock& _block, Chain
             ho.count("number") ? toU256(ho["number"]) : header.number(),
             ho.count("gasLimit") ? toU256(ho["gasLimit"]) : header.gasLimit(),
             ho.count("gasUsed") ? toU256(ho["gasUsed"]) : header.gasUsed(),
-            ho.count("timestamp") ? toU256(ho["timestamp"]) : header.timestamp(),
+            ho.count("timestamp") ? max<u256>(toU256(ho["timestamp"]), 0) :
```

Comment:
`u256` is unsigned, so I don't see the point of `max<u256>(toU256(ho["timestamp"]), 0)`